tests/by-util/test_hashsum.rs

```diff
@@ -29,6 +29,15 @@ macro_rules! test_digest {
                        get_hash!(ts.ucmd().arg(DIGEST_ARG).arg(BITS_ARG).pipe_in_fixture("input.txt").succeeds().no_stderr().stdout_str()));
         }
 
+        #[test]
+        fn test_nonames() {
```

Comment:
Could you also add a test with different files at the same time ?

ex
hashsum --no-names input1.txt input2.txt

Reply:
Yes, how about covering the special `-` at the same time, i.e., `hashsum --no-names input.txt -` with `input.txt` also piped through stdin? I've pushed this now.